### PR TITLE
feat(handlers,glide): implement the createKubeConfigFromCluster function

### LINKED ISSUE #18: Implement the function that generates the kubeconfig file
This issue is a follow on to https://github.com/deis/k8s-claimer/pull/16. `createKubeConfigFromCluster` should be implemented.

### PR DESCRIPTION
**Pull request by arschles**
Fixes #18

### CODE REVIEW

File: handlers/gke.go [hunk 1]
```diff
@@ -26,7 +33,39 @@ func findUnusedGKECluster(clusterMap *clusters.Map, leaseMap *leases.Map) (*cont
 	return nil, errUnusedGKEClusterNotFound
 }
 
-func createKubeConfigFromCluster(cluster *container.Cluster) string {
-	// TODO: implement
-	return ""
+func createKubeConfigFromCluster(cluster *container.Cluster) ([]byte, error) {
+	contextName := strings.ToLower(cluster.Name)
+	authInfoName := strings.ToLower(cluster.Name)
```
Comment (bacongobbler) [inline handlers/gke.go]: this seems unnecessary to call `strings.ToLower` twice here. How about just `authInfoName := contextName`?

**Comment by bacongobbler** [comment]
small nitpick, not necessary to merge if feeling :sleepy:

File: handlers/gke.go (hunk 1 above)
Comment (arschles) [inline handlers/gke.go]: @bacongobbler good catch, thanks. fix incoming shortly

Comment (arschles) [inline handlers/gke.go]: Fixed in https://github.com/deis/k8s-claimer/pull/24/commits/f5dfb9697217ccb90d764c7ff91867855ea32b16

**Pull request merged**